_episodes/28.md

```diff
@@ -0,0 +1,257 @@
+specific routing keys become more popular. To provide a clearer example, say
+that the routing keys are actually just hash geo location values for a taxi app
+that are mapped between zero and one. As certain locations become crowded, lets
+say that a lot of people are going home for the work day, and many taxis are in
+the downtown location. The locations mapped to the downtown routing keys can
+automatically trigger a split, and once the rush hour is over, it will merge
+these segments as traffic slows down. 
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/28/segment-split-merge.png"/><br/>
+Pravega Docs: <a href="https://pravega.io/docs/nightly/pravega-concepts/#elastic-streams-auto-scaling">Source</a>.
+</p>
+
+### Pravega Architecture
+
+The Pravega Architecture comes with writers groups and reader groups that scale
+up and down along with the autoscaling applied to the segments. It consists of
+a controller that maintains stream metadata and the segment store that works off
+of tier one storage (Apache Bookkeeper) and tier two storage (Object storage).
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/28/pravega-architecture.png"/><br/>
+Pravega Docs: <a href="https://pravega.io/docs/nightly/pravega-concepts/#elastic-streams-auto-scaling">Source</a>.
+</p>
+
+Just like Trino, Pravega also aims to build a rich set of connectors with
+systems that will be the source and sink. This includes a connector used for 
+Trino.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/28/pravega-connectors.png"/><br/>
+Pravega Docs: <a href="https://pravega.io/docs/nightly/pravega-concepts/#elastic-streams-auto-scaling">Source</a>.
+</p>
+
+### Pravega compared to other event streaming platforms.
+
+This chart is very helpful resource to summarize Pravega against other popular 
+streaming platforms. This comes from the Pravega site so be sure to check for
+an up to date list of these features moving forward.
+
+|                                    | Pravega | Kafka | Pulsar |
+|------------------------------------|---------|-------|--------|
+| Transactions                       | ✅ | ✅ | ✅ |
+| Event streams                      | ✅ | ✅ | ✅ |
+| Long-term retention                | ✅ |    | ✅ |
+| Durable by default                 | ✅ |    | ✅ |
+| Auto-scaling                       | ✅ |    |    |
+| Ingestion of large data (video)    | ✅ |    |    |
+| efficient at high partition counts | ✅ |    |    |
+| Consistent state replication       | ✅ |    |    |
+| Key-value tables                   | ✅ |    |    |
+
+Comparison between Pravega, Kafka, and Pulsar: [Source](https://pravega.io)
+
+## Demo of the week: Querying Pravega from Trino
+
+This week, Pravega will be showing us a demonstration from their own [getting-started](https://github.com/pravega/presto-connector/tree/main/getting-started)
```

Comment:
This week the Pravega teams demonstrates an example from their getting started tutorial for the Trino connector.